packages/storybook-addon/src/global.d.ts

```diff
@@ -0,0 +1 @@
+declare const __pkg_version__: string;
```

Comment:
unrelated change?

Reply:
it broke without it 😅 something in the storybook addon is really flakey when it comes to the devtools exchange